assets/scss/base/_buttons.scss

```diff
@@ -60,6 +65,10 @@ button, .button {
   color: $link-colour;
   border: none;
 
+  &.button--link-black {
```

Comment:
The "button" is presented as a link as it actually posts to a new page (confirm-deletion). UX want it to be black.